src/index.js

```diff
@@ -933,15 +935,17 @@ class PhoneInput extends React.Component {
     });
     const inputFlagClasses = `flag ${selectedCountry && selectedCountry.iso2}`;
 
+    const Input = this.props.component ? this.props.component : DefaultInput;
```

Comment:
Whenever there's a custom component prop, use it as Input, else, use the default barebones `input` (Which will receive the default className later on L948)